ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -558,14 +560,12 @@ public static void handleInstallDirPermission() {
         try {
             String installationPath = OSUtils.getInstallationPath();
             if (!new File(installationPath).canWrite()) {
-                throw ErrorUtil.createCommandException("current user does not have permissions to run the command\n\n" +
-                                                               "Current user does not have write permissions to " +
-                                                               "Ballerina installation path " + installationPath +
-                                                               ". Grant permission to the file path or run the " +
-                                                               "command with user has write permission.");
+                throw ErrorUtil.createCommandException(
+                        "permission denied: current user does not have write access to the Ballerina installation " +
```

Comment:
We had something similar to these. But there was requirement to tell what we need to do after saying permission have permission. Any +1 revert to short message.

Reply:
@Kishanthan ; Any concerns?

Reply:
A developer would know what to do if we say that the user doesn't have write access to the Ballerina installation directory right?